packages/react/package.json

```
@@ -21,7 +21,7 @@
     "start": "rushx build:icons && rushx storybook",
     "lint": "eslint . -c .eslintrc.json",
     "lint-fix": "eslint --fix . && exit 0",
-    "prepublishOnly": "rushx build",
+    "prepublishOnly": "gulp",
```

why change this??
```suggestion
    "prepublishOnly": "rushx build",
```
or `gulp prod`